packages/ringcentral-widgets/components/ActiveCallPanel/MergeInfo.js

```diff
@@ -26,8 +30,8 @@ function MergeInfo({
         </div>
       </div>
       <div className={styles.merge_item_active}>
-        <div className={styles.callee_avatar_active} style={currentCall.fallBackName !== 'Unknow' ? { backgroundImage: `url(${avatar})` } : { background: '#fff' }}>
-          { currentCall.fallBackName === 'Unknow' || !avatar ? <DefaultAvatar className={styles.defaut_avatar} /> : null}
+        <div className={styles.callee_avatar_active} >
+          { currentCall.fallBackName === 'Unknow' || !currentCall.avatarUrl ? <CallAvatar avatarUrl={null} /> : <CallAvatar avatarUrl={currentCall.avatarUrl} /> }
```

Comment:
currentCall.fallBackName === 'Unknow'
what if the fallBackName change to another string